pyproject.toml

```diff
@@ -3,24 +3,39 @@ requires = ["hatchling"]
 build-backend = "hatchling.build"
 
 [project]
-name = "gcp_scanner"
-description = "Google GCP Scanner"
+name = "gcp-scanner"
+description = "GCP Scanner is a tool that provides insights into the access levels of various GCloud credentials"
```

Comment:
Should we make this consistent with the README?

Reply:
We should also add it to the project description too. 
And some tags like `gcp`, `python`, `security-tools`, `security`.

Reply:
I updated it to align with the readme.